
# HG changeset patch
# User Stephan Bosch <stephan@rename-it.nl>
# Date 1414266676 -7200
# Node ID 15b2910b145c5e16fb9967d16387fc24104b0925
# Parent  819b8fb22034a761b45c88f021a1e8b857ccacb4
Adjusted to datastack-related changes in Dovecot lib-storage.

diff -r 819b8fb22034 -r 15b2910b145c src/lib-sieve/util/edit-mail.c
--- a/src/lib-sieve/util/edit-mail.c	Sun Oct 19 22:50:23 2014 +0200
+++ b/src/lib-sieve/util/edit-mail.c	Sat Oct 25 21:51:16 2014 +0200
@@ -1473,7 +1473,7 @@
 				(&edmail->wrapped->mail, field_name, decode_to_utf8, value_r);
 		}
 
-		t_array_init(&header_values, 1);
+		p_array_init(&header_values, edmail->mail.pool, 1);
 		(void)array_append_space(&header_values);
 		*value_r = array_idx(&header_values, 0);
 		return 0;
@@ -1489,7 +1489,7 @@
 	}
 
 	/* Fill result array */
-	t_array_init(&header_values, 32);
+	p_array_init(&header_values, edmail->mail.pool, 32);
 	field_idx = header_idx->first;
 	while ( field_idx != NULL ) {
 


